pyclient/README.md

```diff
@@ -0,0 +1,61 @@
+
+# Deephaven Python Client 
+
+Deephaven Python Client is a Python package by Deephaven Data Labs. It is a client API that allows Python applications to remotely access Deephaven data servers.
+
+## Source Directory
+
+### From the deephaven-core repository root
+``` shell
+$ cd pyclient
+```
+## Dev environment setup
+``` shell
+$ pip3 install -r pyclient/requirements.txt
+```
+
+## Build
+``` shell
+$ python3 setup.py
+```
+## Run tests
+``` shell
+$ python3 -m unittest discover tests
```

Comment:
Tests are failing for me, not sure if expected:

```
FAILED (failures=3, errors=2)
```

may be due to the version of the server I'm running?

Reply:
no, these are real issues on the server , I have filed tickets for them, but don't think they will be worked on soon. Maybe it is best to comment out these test cases for now.